Description: Don't use --std=c89
 This patch fixes implicit pointer conversions by disabling the c89 GCC option
Author: Roland Stigge <stigge@antcom.de>
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=364544

--- dia-0.97.1.orig/configure.in
+++ dia-0.97.1/configure.in
@@ -411,7 +411,6 @@ if test "$GCC" = yes; then
 
   DIA_CHECK_CFLAG(-Wall)
   dnl DIA_CHECK_CFLAG(-Wunused)
-  DIA_CHECK_CFLAG(--std=c89)
   DIA_CHECK_CFLAG(-Wdeclaration-after-statement)
   dnl I'd rather like to enable this but it fails on isnan() with -std=c89
   dnl DIA_CHECK_CFLAG(-Werror-implicit-function-declaration)

